lib/tictactoe/game.rb

```diff
@@ -4,6 +4,11 @@ module Tictactoe
     
     class Game 
     end
-    class Board 
+    class Board
```

Comment:
I would pull out this class to a separate file. By convention, it would be placed in: `lib/tictactoe/board.rb`